src/Seq.App.EmailPlus/DirectMailGateway.cs

```diff
@@ -1,15 +1,28 @@
 ﻿using System;
-using System.Net.Mail;
+using System.Threading.Tasks;
+using MailKit.Net.Smtp;
+using MimeKit;
 
 namespace Seq.App.EmailPlus
 {
     class DirectMailGateway : IMailGateway
     {
-        public void Send(SmtpClient client, MailMessage message)
+        public async Task<MailResult> Send(SmtpClient client, SmtpOptions options, MimeMessage message)
         {
             if (client == null) throw new ArgumentNullException(nameof(client));
             if (message == null) throw new ArgumentNullException(nameof(message));
-            client.Send(message);
+            try
+            {
+                await client.ConnectAsync(options.Server, options.Port, options.UseSsl);
```

Comment:
Since we connect and disconnect here, any reason not to just create and release a new `SmtpClient` each time?

Reply:
Yeah agree. MailKit initialises quite quickly with minimal overhead and is Async to boot, so that shouldn't be a problem. There are some cases where it's more efficient to hold a single instance of a client (not MailKit) to reduce initialisation overhead, but this was a change before I really got deep into the conversion and proved unnecessary.

Reply:
Now implementing SmtpClient each time. This is now redundant in EmailApp and have removed it from IMailGateway/DirectMailGateway/etc